src/main/steps/applicant/all-documents/start/content.ts

```diff
@@ -0,0 +1,77 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+
+const en = {
+  section: 'Keeping your contact details private',
```

Comment:
Why are we keeping this 'Keeping your contact details private' section content.ts file in 'all-documents' directory?